e2e-tests/src/npg/psp-sorting.npg.integration.test.ts

```diff
@@ -0,0 +1,106 @@
+import { choosePaymentMethod, fillEmailForm, fillOnlyCardDataForm, fillPaymentNotificationForm } from "./helpers";
+import { selectLanguage } from "../verify/helpers";
+
+/**
+ * Test input and configuration
+ */
+const CHECKOUT_URL = String(process.env.CHECKOUT_URL);
+const VALID_FISCAL_CODE = String(process.env.VALID_FISCAL_CODE);
+const EMAIL = String(process.env.EMAIL);
+
+/**
+ * Increase default test timeout (120000ms)
+ * to support entire payment flow
+  */
+
+const timeout = 80_000
+jest.setTimeout(timeout);
+jest.retryTimes(2);
+page.setDefaultNavigationTimeout(timeout);
+page.setDefaultTimeout(timeout)
+
+beforeAll( async () => {
+  await page.goto(CHECKOUT_URL);
+  await page.setViewport({ width: 1200, height: 907 });
+})
+
+beforeEach(async () => {
+  await page.goto(CHECKOUT_URL);
+  await selectLanguage("it");
+});
+
+
+describe("Checkout show and sort psp list", () => {
```

Comment:
Please add a test to check psp name sorting too

Reply:
done here 8d082a44b1b6877b2603d04c36379983f52b8a8c